test/extended/crc/cmd/cmd.go

```diff
@@ -15,6 +15,12 @@ const (
 	CRCExecutableNotInstalled = "notInstalled"
 )
 
+type Command struct {
```

Comment:
Good! In the future we could have a builder like `crc("setup").WithoutNTP().WithoutUpdateCheck()`.